scripts/ast-patches/allowAnyPropOnPlayableMixinNext.js

```diff
@@ -0,0 +1,27 @@
+const ts = require("typescript");
+
+function allowAnyPropOnPlayableMixinNext(ast) {
+  const $w = ast.statements.find(
+    statement =>
+      ts.isModuleDeclaration(statement) && statement.name.escapedText === "$w"
+  );
+  const playableMixin = $w.body.statements.find(
+    statement =>
+      ts.isInterfaceDeclaration(statement) &&
+      statement.name.escapedText === "PlayableMixin"
+  );
+  const playableMixinNext = playableMixin.members.find(
+    member => member.name.escapedText === "next"
```

Comment:
you probably should handle `previous` members as well

Reply:
will do, thanks